### PR TITLE
[ESS Release MS-48] Update docs for release - MS-47 -> MS-48

### PR DESCRIPTION
**Pull request by Z3r0Sum**
This PR changes ms-47 references to ms-48 for the ESS release.

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -721,8 +721,8 @@ contents:
             prefix:     en/cloud-enterprise
             tags:       CloudEnterprise/Reference
             subject:    ECE
-            current:    ms-47
-            branches:   [ {'ms-47': 2.7}, {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
+            current:    ms-48
+            branches:   [ {'ms-48': 2.7}, {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
```
Comment (nrichers) [inline conf.yaml:725]: This is also incorrect, ECE 2. 7 docs need to get built out of MS 47.

File: conf.yaml [hunk 2]
```diff
@@ -721,8 +721,8 @@ contents:
             prefix:     en/cloud-enterprise
             tags:       CloudEnterprise/Reference
             subject:    ECE
-            current:    ms-47
-            branches:   [ {'ms-47': 2.7}, {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
+            current:    ms-48
```
Comment (nrichers) [inline conf.yaml:724]: Current for ECE needs to remain at MS 47, as that's what the codebase is.

File: conf.yaml [hunk 3]
```diff
@@ -738,7 +738,7 @@ contents:
                 repo:   cloud-assets
                 path:   docs
                 map_branches: &mapCloudEceToCloudAssets
-                  ms-47: master
+                  ms-48: master
```
Comment (nrichers) [inline conf.yaml:741]: Also revert this one, please.

File: conf.yaml [hunk 4]
```diff
@@ -753,7 +753,7 @@ contents:
                 repo:   clients-team
                 path:   docs/examples/elastic-cloud/php
                 map_branches: &mapCloudEceToClientsTeam
-                  ms-47: master
+                  ms-48: master
```
Comment (nrichers) [inline conf.yaml:756]: And this one, too.

**Review by nrichers — CHANGES_REQUESTED** [review]
The ESS changes look fine, but you need to revert the changes for the ECE docs. ECE 2.7 is based off MS 47 and the doc build needs to continue using that milestone. I left some related comments.

**Comment by Z3r0Sum** [comment]
> The ESS changes look fine, but you need to revert the changes for the ECE docs. ECE 2.7 is based off MS 47 and the doc build needs to continue using that milestone. I left some related comments.

Sorry about that! Complete.

**Review by nrichers — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**